activation/activation.go

```diff
@@ -238,17 +260,21 @@ func (b *Builder) StopSmeshing(deleteFiles bool) error {
 }
 
 // SmesherID returns the ID of the smesher that created this activation.
-func (b *Builder) SmesherID() types.NodeID {
-	return b.signer.NodeID()
+func (b *Builder) SmesherIDs() []types.NodeID {
+	res := make([]types.NodeID, 0, len(b.signers))
+	for _, sig := range b.signers {
+		res = append(res, sig.NodeID())
+	}
+	return res
```

Comment:
You should lock the `b.smeshingMutex` here. Besides, it could be simplified to:
```suggestion
	return maps.Keys(b.signers)
```